Library/Homebrew/cask/pkg.rb

```diff
@@ -135,6 +135,8 @@ def special?(path)
 
     sig { params(path: T.any(Pathname, T::Array[Pathname])).void }
     def rmdir(path)
+      return unless path.exist?
```

Comment:
This check also needs to be inside the `RMDIR_SH` helper script otherwise it won't be done as `sudo`.

Reply:
Good point. This way, the syntax checker will become green again, too.